src/EditorFeatures/Core/Tagging/ITaggerEventSource.cs

```diff
@@ -29,6 +29,18 @@ internal interface ITaggerEventSource
         /// </summary>
         void Disconnect();
 
+        /// <summary>
+        /// Pauses this event source and prevents it from firing the <see cref="Changed"/> event. Can be called many
+        /// times (but subsequence calls have no impact if already paused).  Must be called on the UI thread.
+        /// </summary>
+        void Pause();
```

Comment:
This interface is implemented by TS, this change break them. FYI @dibarbet 
http://index/?leftProject=Microsoft.CodeAnalysis.EditorFeatures&leftSymbol=nawphull0qjk&file=Tagging%5CITaggerEventSource.cs